### PR TITLE
Add notice about https://github.com/stackabletech/hdfs-operator/issues/274 containing a breaking change to the release notes

### PR DESCRIPTION
**Pull request by soenkeliebau**

### CODE REVIEW

**Review by adwk67 — APPROVED** [review]
lgtm

File: modules/ROOT/pages/release_notes.adoc [hunk 1]
```diff
@@ -163,6 +163,54 @@ to
...
+Previous PVCs were called `hdfs-datanode-default-0` but now need to have the prefix `data-` added, because otherwise there may be naming collisions due to the ability to specify multiple storage classes.
+
+In order to move over existing PVCs a few migration steps are required.
+Since it is not possible to rename PVCs you'll need to delete the existing PVCs and recreate them with the correct name and bound to the correct backing PV.
+
+Please find an example workflow of how this can be achieved below, this also sets the reclaim policy for the backing PV to `Retain` to avoid the PV being deleted when it becomes unbound, depending on your Kubernetes config this step may not be necessary.
+
+[source,bash]
+----
+export PVNAME=$(kubectl get pvc hdfs-datanode-default-0 -o yaml | yq '.spec.volumeName')
```
Comment (maltesander) [inline modules/ROOT/pages/release_notes.adoc:179]: I would mention the yq version. For me on 4.17 i have to do (this solution requires 4.18+):
`export PVNAME=$(kubectl get pvc hdfs-datanode-default-0 -o yaml | yq e '.spec.volumeName' -)`
and maybe add the namespace like:
`export PVNAME=$(kubectl get -n <namespace> pvc hdfs-datanode-default-0 -o yaml | yq '.spec.volumeName')`

**Review by maltesander — COMMENTED** [review]
Should we mention to "stop" the datanodes by setting replicas to 0?

File: modules/ROOT/pages/release_notes.adoc (hunk 1 above)
Comment (fhennig) [reply modules/ROOT/pages/release_notes.adoc:179]: Well, too late

**Pull request merged**